tests/linearGaussian_snle_test.py

```diff
@@ -368,9 +369,11 @@ def test_c2st_multi_round_snl_on_linearGaussian_vi(num_trials: int):
         ("fKL", "gaussian"),
         ("IW", "gaussian"),
         ("alpha", "gaussian"),
+        ("importance", "uniform"),
+        ("importance", "gaussian"),
     ),
 )
-@pytest.mark.parametrize("init_strategy", ("proposal", "sir"))
+@pytest.mark.parametrize("init_strategy", ("proposal", "resample"))
```

Comment:
we should also test for `sir`.

Reply:
should be the same when using the prior, no? but anyway.

Reply:
See comment above: it's only the same if the prior is uniform